content/themes/casper

```diff
@@ -1 +1 @@
-Subproject commit 1c092c7be098bed496eaa2f194a3ae39dd23659d
+Subproject commit ede6b03afad148eadba701bce9e65401d9e034e8
```

Comment:
Can you rebase and ensure that your submodules are up to date or remove this change from the commit?